assets/prototype/application/ui/components/display/espressoIcon/index.ts

```diff
@@ -1,3 +1,3 @@
 export { default as EspressoIcon } from './EspressoIcon';
 export { default as isEspressoIcon } from './isEspressoIcon';
-export { EspressoIconProps, Icon } from './types';
+export * from './types';
```

Comment:
Fix for: 
![image](https://user-images.githubusercontent.com/1477453/72339863-c6d1f480-36cf-11ea-96c3-46f2cdbf29af.png)

Reply:
![image](https://user-images.githubusercontent.com/1477453/72427635-da935e80-3794-11ea-8925-7f3f697fddfb.png)